static/script.js

```diff
@@ -24,7 +24,7 @@ $(function() {
     // styles for map
     // https://developers.google.com/maps/documentation/javascript/styling
     var styles = [
-        // hide Google's labels
+     /*   // hide Google's labels
```

Comment:
We are not removing this functionality. We have to make other label's invisible. You can't simply comment out this piece of code.

Reply:
We don't want labels of parks, museums etc.  visible